package.json

```diff
@@ -61,7 +63,7 @@
     "build": "npm run build-client && npm run build-server",
     "build-css": "node_modules/.bin/node-sass public/scss/application.scss public/dist/application.css",
     "build-client": "npm run clean-client && node_modules/.bin/webpack --config webpack.config.prod.js && npm run build-css",
-    "build-server": "npm run clean-server && node_modules/.bin/babel ./ -d ./build -s --copy-files --ignore node_modules/,test/,seeders/,migrations/,build/ && cp -r node_modules/ ./build/node_modules",
+    "build-server": "npm run clean-server && node_modules/.bin/babel ./ -d ./build -s --copy-files --ignore test/,seeders/,migrations/,build/",
```

Comment:
We should be ignoring node_modules on babel and copy it directly. Otherwise your installation will fail since you are applying babel to the node_modules folder.